Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -197,12 +198,10 @@ final class NetworkProtectionConnectionTester {
             self?.timer = nil
         }
 
-        isRunning = true
         timer.resume()
     }
 
     private func stopScheduledTimer() async {
-        isRunning = false
```

Comment:
This is only called internally and should not signal that the monitor isn't running.

Reply:
:+1: